index.html

```diff
@@ -0,0 +1,239 @@
+                alt="Placeholder image"
+              />
+            </div>
+            <p class="card-description">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit.
+              Cupiditate architecto quo voluptatum, quisquam alias illum porro,
+              maiores quam modi rerum soluta incidunt ipsum cumque minus, fugit
+              facere non inventore saepe.
+            </p>
+          </div>
+
+          <!--This is the section containing a project card with a title, an image and a description-->
+          <div class="secondary-cards">
+            <h3 class="project-title">Title</h3>
+            <div>
+              <img
+                class="card-img"
+                src="https://via.placeholder.com/180x65"
+                alt="Placeholder image"
+              />
+            </div>
+            <p class="card-description">
+              Lorem ipsum dolor sit amet consectetur, adipisicing elit. Veniam
+              sint perferendis repudiandae voluptates officia quibusdam eum.
+              Consequatur, earum, et dignissimos ullam possimus corrupti aperiam
+              delectus, eum dolorem provident deserunt iste?
+            </p>
+          </div>
+
+          <!--This is the section containing a project card with a title, an image and a description-->
+          <div class="secondary-cards">
+            <h3 class="project-title">Title</h3>
+            <div>
+              <img
+                class="card-img"
+                src="https://via.placeholder.com/180x65"
+                alt="Placeholder image"
+              />
+            </div>
+            <p class="card-description">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsam
+              vero laudantium possimus voluptatem ipsa eaque non iure, aperiam
+              ipsum molestiae. Consequatur ullam quia asperiores velit ipsa
+              sequi laboriosam consequuntur id.
+            </p>
+          </div>
+
+          <!--This is the section containing a project card with a title, an image and a description-->
+          <div class="secondary-cards">
+            <h3 class="project-title">Title</h3>
+            <div>
+              <img
+                class="card-img"
+                src="https://via.placeholder.com/180x65"
+                alt="Placeholder image"
+              />
+            </div>
+            <p class="card-description">
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit. Ad omnis
+              ipsam alias in. Minus corrupti dolore provident non suscipit
+              doloremque necessitatibus, ab est quisquam totam voluptatum sequi,
+              in earum impedit?
+            </p>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <!--This is the Footer containing the Contacts section -->
+    <footer class="footer" id="contact">
+      <ul class="contacts-list">
+        <li class="contact">Email</li>
```

Comment:
Please add in the contact info here! Check out the `mailto` option on the `href` attribute to allow the link to directly open up the user's email. You can also add your linkedin link too.

Reply:
I will add in the future as i dont have a Linkedin yet and i dont wanna expose my email for now.

thanks